### ISSUE TITLE
EXITING because of FATAL problem while generating the suffix array

### ISSUE THREAD

**Issue by realDongWang** [alexdobin/STAR#534]
Hello Alex,
I used RSEM and STAR to convert Fastq files to gene expression data. However, I got some problems. There is the error code:

EXITING because of FATAL problem while generating the suffix array 87 The number of indices read from chunks = 4630099134 is not equal to expected nSA=5729570440
SOLUTION: try to re-run suffix array generation, if it still does not work, report this problem to the author
Dec 11 22:12:11 ...... FATAL ERROR, exiting
"STAR --runThreadN 4 --runMode genomeGenerate --genomeDir . --genomeFast aFiles /home/wangdong/Data/GRCh/Homo_sapiens.GRCh37.68.dna.primary_assembly. fa --sjdbGTFfile /home/wangdong/Data/GRCh/Homo_sapiens.GRCh37.68.gtf --sjd bOverhang 100 --outFileNamePrefix ./human_ensembl" failed! Plase check if y ou provide correct parameters/options for the pipeline!

Thanks in advance
Dong

**Comment by alexdobin** [comment]
Hi @realDongWang 

this is most likely due to running out of space on your hard disk, you need ~100GB .
If this does not solve the problem, please send me the Log.out file.

Cheers
Alex

**Comment by realDongWang** [comment]
> Hi @realDongWang
> 
> this is most likely due to running out of space on your hard disk, you need ~100GB .
> If this does not solve the problem, please send me the Log.out file.
> 
> Cheers
> Alex

Thanks for your help!
I fixed this problem. But I met a new problem. There is the error code:

EXITING because of FATAL ERROR: Read1 and Read2 are not consistent, reached the end of the one before the other one
SOLUTION: Check you your input files: they may be corrupted

I downloaded some SRA data (paired-end) from NCBI. I wonder if I need to convert them into two fastq files or just one fastq file.
